### PR TITLE
Add #bs-alert to handbook

### PR DESCRIPTION
**Pull request by brianjgeiger**
## Purpose

Make our docs slightly more complete with the `#bs-alert` component.

## Summary of Changes

Add a bs-alert component with a couple of examples and a link to the docs

bs-alert-more-truthful

## Side Effects / Testing Notes

Shouldn't have too many side effects, really. Test with the handbook enabled on localhost:4200/handbook/docs/

## Ticket

https://openscience.atlassian.net/browse/EMB-

# Reviewer Checklist

- [ ] meets requirements
- [ ] easy to understand
- [ ] DRY
- [ ] ~~testable and includes test(s)~~
- [ ] ~~changes described in `CHANGELOG.md`~~

### CODE REVIEW

File: lib/handbook/addon/docs/components/bs-alert/template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# bs-alert
+
+Generates a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html).
+
+## info
+{{docs/components/bs-alert/demo-info}}
+
+## warning (not dismissible)
+{{docs/components/bs-alert/demo-warn-no-dismiss}}
```
Comment (jamescdavis) [inline lib/handbook/addon/docs/components/bs-alert/template.md]: Maybe also add an example demonstrating `hidden` for completeness?

Something like:

```handlebars
{{#docs-demo as |demo|}}
    {{#demo.example name='bs-alert.hidden-danger.hbs'}}
            {{#bs-alert type='danger' dismissible=false hidden=hideAlert}}
                    <strong>Danger, Will Robinson!</strong>
            {{/bs-alert}}
    {{/demo.example}}

    {{#bs-button class='center-block m-b-lg' onClick=(action (mut hideAlert) (not hideAlert))}}
        set: hideAlert = {{not hideAlert}}
    {{/bs-button}}

    {{demo.snippet 'bs-alert.hidden-danger.hbs'}}
{{/docs-demo}}
```
would work nicely.

File: lib/handbook/addon/docs/components/bs-alert/template.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# bs-alert
+
+Generates a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html).
```
[the remark this answers is not in the document]
Comment (brianjgeiger) [reply lib/handbook/addon/docs/components/bs-alert/template.md]: Is it just that we use a font-awesome icon for dismissing? I don't see a lot else going on in our component.

[the remark this answers is not in the document]
Comment (brianjgeiger) [reply lib/handbook/addon/docs/components/bs-alert/template.md]: Oh, or also the hidden stuff, below.

**Comment by brianjgeiger** [comment]
@jamescdavis Updated

File: lib/handbook/addon/docs/components/bs-alert/template.md [hunk 3]
```diff
@@ -1,9 +1,16 @@
 # bs-alert
 
-Generates a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html).
+Generates what is mostly a [boostrap alert](https://www.ember-bootstrap.com/api/classes/Components.Alert.html). 
+
+In addition to the standard look and functionality, we:
+ - Have a font-awesome close icon to match OSF styling;
+ - Have the ability to hide and show the alert.
```
Comment (jamescdavis) [inline lib/handbook/addon/docs/components/bs-alert/template.md]: The ability to hide is actually something [ember-bootstrap has too](https://github.com/kaliber5/ember-bootstrap/blob/master/addon/templates/components/common/bs-alert.hbs#L1).

The other thing we do add, though, is [i18n](https://github.com/CenterForOpenScience/ember-osf-web/blob/develop/lib/osf-components/addon/components/bs-alert/template.hbs#L3).

File: lib/handbook/addon/docs/components/bs-alert/demo-danger-hidden/template.hbs [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+                    <strong>Danger, Will Robinson!</strong>
+            {{/bs-alert}}
+        {{/demo.example}}
+
+        {{#bs-button class='center-block m-b-lg' onClick=(action (mut hideAlert) (not hideAlert))}}
+            set: hideAlert = {{not hideAlert}}
+        {{/bs-button}}
+
+    {{demo.snippet 'bs-alert.danger-hidden.hbs'}}
+{{/docs-demo}}
```
Comment (jamescdavis) [inline lib/handbook/addon/docs/components/bs-alert/demo-danger-hidden/template.hbs]: Thank you for indulging me!

**Comment by brianjgeiger** [comment]
Updated again with 10% fewer lies.

**Pull request merged**